docs/expressions/expression_trees.md

```diff
@@ -39,10 +39,11 @@ tree generated by `math.parse('sqrt(2 + x)')`.
 
 All nodes have the following methods:
 
--   `clone() : Node`
+-   `clone(options: MetaOptions) : Node`
```

Comment:
The `options` are optional, can you describe that here and for all constructors in the docs?